packages/labs/ssr-react/src/test/integration/enable-lit-ssr_test.tsx

```diff
@@ -210,4 +214,67 @@ test('nested element', () => {
   );
 });
 
-test.run();
+bareCEtest.run();
+
+const wrappedCEtest = suite('@lit-labs/react wrapped custom elements');
+
+const ReactTestElement = createComponent({
+  react: React,
+  tagName: 'test-element',
+  elementClass: TestElement,
+});
+
+wrappedCEtest('wrapped element without prop', () => {
+  assert.equal(
+    ReactDOMServer.renderToString(<ReactTestElement />),
+    `<test-element><template shadowroot="open" shadowrootmode="open"><style>
+    p {
+      color: blue;
+    }
+  </style><!--lit-part aHUgh01By8I=--><p>Hello, <!--lit-part-->Somebody<!--/lit-part-->!</p>
+      <slot></slot><!--/lit-part--></template></test-element>`
+  );
+});
+
+wrappedCEtest('wrapped element with prop', () => {
+  assert.equal(
+    ReactDOMServer.renderToString(<ReactTestElement name="React" />),
+    `<test-element defer-hydration=""><template shadowroot="open" shadowrootmode="open"><style>
+    p {
+      color: blue;
+    }
+  </style><!--lit-part aHUgh01By8I=--><p>Hello, <!--lit-part-->React<!--/lit-part-->!</p>
+      <slot></slot><!--/lit-part--></template></test-element>`
+  );
+});
+
+wrappedCEtest('wrapped element with prop and attribute', () => {
+  assert.equal(
+    ReactDOMServer.renderToString(
+      <ReactTestElement name="React" id="react-test-element" />
+    ),
+    `<test-element id="react-test-element" defer-hydration=""><template shadowroot="open" shadowrootmode="open"><style>
+    p {
+      color: blue;
+    }
+  </style><!--lit-part aHUgh01By8I=--><p>Hello, <!--lit-part-->React<!--/lit-part-->!</p>
```

Comment:
Are the lit part keys stable? In this case `aHUgh01By8I`? Can these be stripped from test using some equivalent of the `stripExpressionMarkers` helpes?

Reply:
That part is a digest built from the static portion of the template so it's stable as long as the template doesn't change. These digests are used client side during hydration to detect any changes. I don't feel the need to strip them as I consider these more snapshot tests of the SSRed result. I do think there's room to improve the test infrastructure to more easily generate/update these assertions but haven't gotten around to that. :/